$OpenBSD: patch-src_parser_h,v 1.1 2019/01/18 09:00:15 fcambus Exp $

Fix out-of-bounds access.
Upstream commit c51f4c2f92288f923cf33bdc395501f447fe2d5c.

Index: src/parser.h
--- src/parser.h.orig
+++ src/parser.h
@@ -57,7 +57,7 @@ class Writer (private)
 	wofstream file;
 	ToDo& todo;
 	iToDo* i;
-	char path[128];
+	char path[256];
 
 	void _save();
 	void amp(wstring& str);
